manubot/cite/handlers.py

```diff
@@ -358,35 +539,86 @@ def get_csl_item(self, citekey) -> Dict[str, Any]:
     "ega.dataset": "manubot.cite.curie.Handler_CURIE",
     "ega.study": "manubot.cite.curie.Handler_CURIE",
     "eggnog": "manubot.cite.curie.Handler_CURIE",
+    "egid": "manubot.cite.curie.Handler_CURIE",
+    "ehda": "manubot.cite.curie.Handler_CURIE",
+    "ehdaa": "manubot.cite.curie.Handler_CURIE",
+    "ehdaa2": "manubot.cite.curie.Handler_CURIE",
+    "ehdaa2_retired": "manubot.cite.curie.Handler_CURIE",
     "elm": "manubot.cite.curie.Handler_CURIE",
+    "emap": "manubot.cite.curie.Handler_CURIE",
+    "emapa": "manubot.cite.curie.Handler_CURIE",
+    "emapa_retired": "manubot.cite.curie.Handler_CURIE",
     "emdb": "manubot.cite.curie.Handler_CURIE",
+    "emolecules": "manubot.cite.curie.Handler_CURIE",
+    "empiar": "manubot.cite.curie.Handler_CURIE",
+    "ena": "manubot.cite.curie.Handler_CURIE",
     "ena.embl": "manubot.cite.curie.Handler_CURIE",
     "encode": "manubot.cite.curie.Handler_CURIE",
+    "enm": "manubot.cite.curie.Handler_CURIE",
     "ensembl": "manubot.cite.curie.Handler_CURIE",
     "ensembl.bacteria": "manubot.cite.curie.Handler_CURIE",
     "ensembl.fungi": "manubot.cite.curie.Handler_CURIE",
     "ensembl.metazoa": "manubot.cite.curie.Handler_CURIE",
     "ensembl.plant": "manubot.cite.curie.Handler_CURIE",
     "ensembl.protist": "manubot.cite.curie.Handler_CURIE",
+    "ensemblglossary": "manubot.cite.curie.Handler_CURIE",
+    "entrez": "manubot.cite.curie.Handler_CURIE",
+    "entrez gene/locuslink": "manubot.cite.curie.Handler_CURIE",
```

Comment:
Will whitespace and punctuation in a prefix cause problems anywhere for Manubot?

Reply:
IMO these should be disallowed in Manubot - this will probably cause a lot of havoc with CURIE parsing. These synonyms are in the Bioregistry because other resources are propagating this kind of thing and it provides extended abilities for parsing malformed CURIEs, but Manubot probably shouldn't get into this business nor enable people to do this kind of things

Reply:
Okay so should we filter prefix + synonyms to those matching the proposed prefex regex at https://github.com/biopragmatics/bioregistry/issues/158

> New prefixes must validate against the following regular expression: ^[a-z][a-z0-9]+(\.[a-z][a-z0-9]+?)$

So manubot would only support the subset of prefixes that are not malformed?